packages/vite-plugin/src/main.test.ts

```diff
@@ -547,3 +547,61 @@ defined on your team and site and much more. Run npx netlify init to get started
     })
   })
 })
+
+test('Works with Vite 5', { timeout: 15_000 }, async () => {
```

Comment:
Rather than having a separate test for Vite 5, could we have a matrix and run all tests for all versions of Vite?